projects/angular/src/data/datagrid/datagrid-virtual-scroll.directive.ts

```diff
@@ -237,6 +249,11 @@ export class ClrDatagridVirtualScrollDirective<T> implements AfterViewInit, DoCh
     this._totalItems = value;
   }
 
+  private set fixedItemSize(value: CdkFixedSizeVirtualScrollInputs['itemSize']) {
+    this._cdkFixedSizeVirtualScrollInputs.itemSize = coerceNumberProperty(value);
+    this.updateFixedSizeVirtualScrollInputs();
+  }
```

Comment:
I think it should be a normal function and not a setter as it looks confusing.
```suggestion
  private updateFixedItemSize(value: CdkFixedSizeVirtualScrollInputs['itemSize']) {
    this._cdkFixedSizeVirtualScrollInputs.itemSize = coerceNumberProperty(value);
    this.updateFixedSizeVirtualScrollInputs();
  }
```

Reply:
Correcterd in commit 6db1205